test/abstractarray.jl

```diff
@@ -398,14 +398,17 @@ function test_map(::Type{TestAbstractArray})
     )
         X = typ[1:10...]
         _typ = typeof(arg_typ(one(typ)))
-        @test map(arg_typ, X) == _typ[1:10...]
+        for mapf in [map, asyncmap, pmap]
```

Comment:
In the latest commit, I've removed use of workers from `test/abstractarray.jl` and updated the `JULIA_TESTFULL=1` tests in `test/parallel_exec.jl`.